embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/arch/schema/EmbType.kt

```diff
@@ -92,9 +92,9 @@ public sealed class EmbType(type: String, subtype: String?) : TelemetryType {
             )
         }
 
-        public object ReactNativeAction : System("rn_action", true)
```

Comment:
This was being sent as "true" before, and now it's using the default sendMode. Is this expected?

Reply:
Yes - this is a span, and this doesn't apply anyway. I think it was a mistake to supply it before.